config/site_settings.yml

```diff
@@ -764,6 +764,14 @@ rate_limits:
   tl2_additional_likes_per_day_multiplier: 1.5
   tl3_additional_likes_per_day_multiplier: 2
   tl4_additional_likes_per_day_multiplier: 3
+  alert_admins_if_errors_per_minute:
+    client: true
+    shadowed_by_global: true
+    default: 100
```

Comment:
also we got to think about how we are going to get this all going in our environment, we may need a custom plugin for meta to poll rate, cause error reports are reported from multiple sources.